anesthetic/plot.py

```diff
@@ -148,6 +164,8 @@ class AxesDataFrame(DataFrame):
         If 'outer', plot ticks only on the very left and very bottom.
         If 'inner', plot ticks also in inner subplots.
         If None, plot no ticks at all.
+    logxy : list(str), optional
+        List of parameters to be plotted on a log scale.
```

Comment:
It would be a very unusual set-up but we could in principle allow for some parameters to be log on the x axis and linear on the y axis (in e.g. a rectangular plot).

Reply:
Hui, that would mess with our distinction of `DiagonalAxes` (which are for 1d plots) and `OffDiagonalAxes` (which are for 2d plots)...

Reply:
By #59, we should probably go with `loglog` rather than `logxy`.

If somebody needs the fine-grained control I describe, we can e.g. add tuples to this list as an option at a laterdatae.

Reply:
`loglog` is never really true. `loglog` would be a kwarg used for two different parameters x and y that are both supposed to be on a log axes. Here, we are applying a log axis to a range of plots for a given parameter z and dependent on whether parameter z appears on x or y axis we are going with the `logx` or `logy` kwarg...